src/main/resources/assets/thaumicaugmentation/lang/en_us.lang

```diff
@@ -595,9 +595,16 @@ thaumicaugmentation.research.vis_battery_augment.1=Although it does not feel lik
+thaumicaugmentation.research.thaumium_robes.0=I have had access to robes made of enchanted fabric for a while now, but in hindsight they seem somewhat… §olacking.§r They offer me little to no protection against the creatures that inhabit this world. I cannot let the risk of danger stop my research, but at the same time it would be foolish of me to not be prepared to defend myself. Thankfully, I believe I may be able to enhance my original robes with my recently invented thaumium.<BR>Thaumium’s unique magical conductivity may very well prove beneficial to spell-casting, augmenting the already magically attuned enchanted fabric very effectively. Furthermore, addition of the metal should also provide more protection from attacks, perhaps even from non-physical ones.<BR>I’ll need to plan an appropriate set of infusion recipes to create these new robes, so while I am working on them I may as well improve the robes’ visual design too. After all, any Thaumaturge worth their salt should aim to strike a good balance between functionality, efficiency and style.
 thaumicaugmentation.research.thaumium_robes.1=Thaumium has proven itself to be an excellent metal to work with, meshing well with enchanted fabric. I now have a set of infusion recipes for the Robes of the Adept, including a hood with built-in Goggles of Revealing, a chest-piece and a pair of leggings. I have left out the boots as I feel I may be able to enhance them in a very different way, to allow them to do more than just protect me.<BR>Regardless, these new robes offer far better protection than before, with the additional thaumium plates being able to absorb both physical and magical attacks. Furthermore, the robes also have an improved vis discount on them, allowing me to cast spells and craft magical items just a little more efficiently. They also accept enchantments very well, again due to the thaumium used to make them.<BR>Finally, I have improved the robes’ visual design and I can even recolour them if I so wish. To do this, I simply have to craft the robes with dye(s) in a crafting window, and to wash the dye out I just have to use the robes on a filled, mundane cauldron.
+
+thaumicaugmentation.research.glass_tube.title=Glass Essentia Tube
+thaumicaugmentation.research.glass_tube.0=Essentia piping systems are a major step forward in thaumaturgic alchemy, but this major step forward comes at the price of additional complexity. Managing tube systems can easily become confusing, especially since I cannot easily tell at a glance where and when essentia is flowing, what with all tube types being fully opaque.<BR>This was not done by choice, as previously was not able to find a way to make a pipe primarily made of glass survive the constant barrage of magical essence. However, I may be able to revisit my previous studies and find a way to make glass resistant to even the most volatile of essentia. Perhaps I should look into warded jars as a starting point.
```

Comment:
"major step forward" is repeated in the first sentence, that can probably sound more varied